composer.json

```diff
@@ -23,18 +27,25 @@
         "dflydev/dot-access-data": "^1.1.0 || ^2 || ^3",
         "grasmash/yaml-expander": "^3",
         "loophp/phposinfo": "^1.7.1",
-        "symfony/filesystem": "^6.2",
+        "symfony/filesystem": "^5.4 || ^6.2",
```

Comment:
I think we should include `"symfony/filesystem": "^4.4 || ^5.4 || ^6.2",` then only it can be check for drupal9 and php 7.4

Reply:
+1

Reply:
Support for 4.4 is not needed as 5.4 already gives support for D9 and php 7.4